### ISSUE TITLE
Emoji domain doesn't actually redirect

### ISSUE THREAD

**Issue by shkm** [thedevs-network/kutt#85]
So I've set up an emoji domain with kutt.it and set an A record to 206.189.194.85.

Now here's a short URL that's meant to redirect to Google: http://💪🏻.to/testing

It resolves to http://xn--mn8h3k.to/testing and seems to reach `kutt.it`, but never actually redirects. I assume this is something to do with the routing table not recognising the domain that the emoji is resolved to.

**Comment by shkm** [comment]
If I set up the domain in `kutt.it` using the puny code, however, this does seem to work. As such, the issue isn't as pressing as I'd originally thought, but I'd also expect to be able to enter an emoji domain and have it just work :muscle:.

**Comment by poeti8** [comment]
Yes it's because I'm not handling the emojis thus the domain name that is being save would not match xn--mn8h3k.to. However if you save domain as xn--mn8h3k.to it should work fine.
I'll handle this whenever I got free.

**Comment by hiyukoim** [comment]
> However if you save domain as xn--mn8h3k.to it should work fine.
in my case, I couldn't add my emoji domain from "Custom domain" section, even with the decoded version start with "xn--"
it simply gives me an error message "Couldn't add domain".

**Comment by trgwii** [comment]
This seems to work now (Or browsers autofix it :shrug:)
